custom/polio_rdc/ucr/data_sources/users.json

```diff
@@ -0,0 +1,291 @@
+{
+    "domains": [
+        "polio-rdc"
+    ],
+    "server_environment": [
+        "production"
+    ],
+    "config": {
+        "table_id": "commcare_users",
+        "display_name": "Users",
+        "referenced_doc_type": "CommCareUser",
+        "description": "Data from the User Model",
+        "base_item_expression": {},
+        "configured_filter": {},
+        "validations": [],
+        "configured_indicators": [
+            {
```

Comment:
@idiene are you able to look through this and confirm it has everything you'd need? One obstacle is that we can't reproduce the `name` column without some dev, but I'm working on that.